sdv/_utils.py

```diff
@@ -271,3 +273,25 @@ def _get_rows_to_drop(metadata, data):
             relationships = [rel for rel in relationships if rel not in relationships_parent]
 
     return table_to_idx_to_drop
+
+
+def _validate_foreign_keys(metadata, data):
```

Comment:
so that this doesn't get confused with the `_validate_foreign_keys` method in the [metadata ](https://github.com/sdv-dev/SDV/blob/1a26d18bc73abefcefda6724c96cb5f3a6f579b9/sdv/metadata/multi_table.py#L747)can we call this `_validate_foreign_keys_not_null` or something?